### PR TITLE
add default event for scheduled eventbridge rules

### LINKED ISSUE #5387: bug: EventBridge Scheduled Rules generate an empty event
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When a scheduled rules runs, an empty event is generated and sent to targets.

### Expected Behavior

An event like this sample should be sent.
```json
{
  "version": "0",
  "id": "89d1a02d-5ec7-412e-82f5-13505f849b41",
  "detail-type": "Scheduled Event",
  "source": "aws.events",
  "account": "123456789012",
  "time": "2016-12-30T18:44:49Z",
  "region": "us-east-1",
  "resources": ["arn:aws:events:us-east-1:123456789012:rule/SampleRule"],
  "detail": {}
}
```

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker-compose up

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal events put-rule --name scheduled_rule --schedule-expression "rate(1 minutes)"
    awslocal events put-targets --rule scheduled_rule --targets "Id"="1","Arn"="arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo"
    awslocal--endpoint-url=http://localhost:4566 logs tail "/aws/events/echo"

### Environment

```markdown
- OS: macOS Monterey
- LocalStack: 0.13.3
```

### Anything else?
I think [this](https://github.com/localstack/localstack/blob/1567f25a8f672f4dec23cb7cb1aadb5a744366f8/localstack/services/events/events_listener.py#L92) means an non-empty event is sent only when an input is `Constant (JSON text)`.
image

### PR DESCRIPTION
**Pull request by thrau**
## Motivation

This PR fixes #5387, which required some additional aws validated tests and a little bit of extra logic.

Testing scheduled rules leads to either huge test methods (testing multiple targets in one test), or slow test execution (one test per test target, which will take at least 1 minute each, since that's the minimum schedule rate). To achieve proper test isolation and fix the slow test execution, I added a developer endpoint `/_aws/events/rules/<rule_arn>/trigger` that allows you to trigger a scheduled rule immediately that's running in the internal job scheduler.

The trigger endpoint could be interesting for local testing, and we could add it to [the docs](https://docs.localstack.cloud/user-guide/aws/eventbridge/).

I pretty much re-worked the whole scheduler in the [rework-job-scheduler](https://github.com/localstack/localstack/tree/rework-job-scheduler) branch, and this PR sets up a testing basis for it.

## Changes

* Added a new `scheduled_rule` testing package with one test per service (added sqs and logs for now)
* Moved some re-usable fixtures from `test_events.py` to `conftest.py` in the events test package
* Added `/_aws/events/rules/<rule>/trigger` endpoint to trigger a scheduled rule programmatically
* In the scheduled job func, create a default event
* Fixes #5387

## Future work

* Tests are still a bit rudimentary and we should test `InputPath` and `InputTransformer` for the scheduled events

### CODE REVIEW

**Review by dominikschubert — APPROVED** [review]
Added a few nits regarding the test/fixture setup, otherwise LGTM!

Really like the approach of using internal routes for working around the minimum schedule which should also make the tests less flaky. :+1:

File: tests/aws/services/events/scheduled_rules/test_events_scheduled_rules_logs.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    def _get_events():
+        _events = []
+
+        for log_stream in aws_client.logs.describe_log_streams(logGroupName=logs_log_group)[
+            "logStreams"
+        ]:
+            _response = aws_client.logs.get_log_events(
+                logGroupName=logs_log_group, logStreamName=log_stream["logStreamName"], limit=10
+            )
+            _events.extend(_response.get("events", []))
```
[the remark this answers is not in the document]
Comment (thrau) [reply tests/aws/services/events/scheduled_rules/test_events_scheduled_rules_logs.py:101]: thank you!! i couldn't figure out how to do that

File: tests/aws/services/events/scheduled_rules/conftest.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# @pytest.fixture(autouse=True)
+def _speed_up_localstack_scheduler(monkeypatch):
+    # this hack speeds up the test on localstack, triggering the rule in 5 seconds rather than 1 minute
+    from localstack.services.events import scheduler
+
+    monkeypatch.setattr(scheduler, "parse_rate_expression", lambda e: timedelta(seconds=5))
+
+
+@pytest.fixture
```
[the remark this answers is not in the document]
Comment (thrau) [reply tests/aws/services/events/scheduled_rules/conftest.py:18]: ok, changed it to a util function

File: tests/aws/services/events/conftest.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            sqs_client = aws_client.sqs
+            sqs_client.delete_queue(QueueUrl=queue_url)
+        if log_group_name:
+            logs_client = aws_client.logs
+            log_streams = logs_client.describe_log_streams(logGroupName=log_group_name)
+            for log_stream in log_streams["logStreams"]:
+                logs_client.delete_log_stream(
+                    logGroupName=log_group_name, logStreamName=log_stream["logStreamName"]
+                )
+            logs_client.delete_log_group(logGroupName=log_group_name)
```
[the remark this answers is not in the document]
Comment (thrau) [reply tests/aws/services/events/conftest.py:106]: i didn't actually add this function, but refactored it anyway!

File: tests/aws/services/events/scheduled_rules/test_events_scheduled_rules_logs.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (thrau) [reply tests/aws/services/events/scheduled_rules/test_events_scheduled_rules_logs.py]: i restructured the test to also add a second target into the same log group and used your pattern to get all of the events from all the different log streams. actually found an issue with the implementation this way :grin: thanks again

**Review by dfangl — APPROVED** [review]
LGTM! Really like the backdoor api to trigger scheduled rules, we just have to make sure to keep at least one test which tests the actual scheduling as well now :)

**Pull request merged**